### PR TITLE
Matteo/custom block test

### PR DESCRIPTION
**Pull request by matteobjornsson**
Unmock metrics and add csv printout.

There is still something missing in the unlocked environment, observed in the log outputs when this test is run.

### CODE REVIEW

File: engine/execution/computation/computer/computer.go [hunk 1]
```diff
@@ -119,6 +119,8 @@ func (e *blockComputer) executeBlock(
 	programs *programs.Programs,
 ) (*execution.ComputationResult, error) {
 
+	startedAt := time.Now()
```
Comment (matteobjornsson) [inline engine/execution/computation/computer/computer.go:122]: Turns out the timing info is not coming from the spans but rather pretty basic timing calls. I think the spans are more for the more complex metrics for when the network is actually running.

File: engine/execution/computation/computer/computer.go [hunk 2]
```diff
@@ -226,6 +228,18 @@ func (e *blockComputer) executeBlock(
...
 	res.TrieUpdates = trieUpdates
 
+	elapsedTimeNS := time.Since(startedAt)
+	e.log.Info().Hex("block_id", logging.Entity(blockCtx.BlockHeader)).
+		Int("numberOfTransactions", len(res.TransactionResults)).
+		Int64("timeSpentInNS", elapsedTimeNS.Nanoseconds()).
+		Timestamp().
+		Msg("Block executed")
+
+	msg := "***  Block executed in: " + elapsedTimeNS.String() + "  ***"
```
Comment (matteobjornsson) [inline engine/execution/computation/computer/computer.go:238]: Put this printout in the `computer.go` file for now, but we can move it to the test file if that feels like better organization.

File: engine/execution/computation/computer/computer.go [hunk 3]
```diff
@@ -226,6 +228,18 @@ func (e *blockComputer) executeBlock(
 	res.Proofs = proofs
 	res.TrieUpdates = trieUpdates
 
+	elapsedTimeNS := time.Since(startedAt)
```
Comment (matteobjornsson) [inline engine/execution/computation/computer/computer.go:231]: This is already logged in the execution manager, but at the `computer.go` level we don't see it in the unit tests.

File: engine/execution/computation/computer/customblocks_test.go [hunk 4]
```diff
@@ -105,7 +122,7 @@ func (vmt vmTest) run(
 		ledger.On("Prove", mock.Anything).
 			Return(expectedProof, nil).
 			Times(numTxPerCol*numCol + 1)
-		committer := committer.NewLedgerViewCommitter(ledger, trace.NewNoopTracer())
+		committer := committer.NewLedgerViewCommitter(ledger, tracer)
```
Comment (matteobjornsson) [inline engine/execution/computation/computer/customblocks_test.go:125]: added the unmocked tracer here, not totally sure if it needs its own tracer or not

**Comment by tatujan** [comment]
Discussed and closed this PR on Thursday Meeting.

**Pull request merged**